eng/Publishing.props

```diff
@@ -135,7 +135,13 @@
       <IsStableBuild Condition="'$(DotNetFinalVersionKind)' == 'release'">true</IsStableBuild>
     </PropertyGroup>
 
+    <!-- In VMR inner-build copy to local storage, do not push to AzDO -->
+    <PropertyGroup Condition="'$(ArcadeInnerBuildFromSource)' == 'true' or '$(DotNetBuildPhase)' == 'InnerRepo'">
+      <PushToLocalStorage>true</PushToLocalStorage>
+    </PropertyGroup>
```

Comment:
Why is that set here instead of in Arcade's publishing targets? Isn't that the default for all repos, not just installer now?

Reply:
You're correct. I will set this property early in `arcade` (publish.proj), before we import Publishing.props, and get rid of this.

Good idea.

Reply:
Fixed with https://github.com/dotnet/installer/pull/18917/commits/9ccd9fef91977119e248048e95b7178e06691a86